site/docs/v1/tech/apis/applications.adoc

```diff
@@ -9,6 +9,7 @@ description: The APIs for creating, retrieving, updating and deleting Applicatio
 This page contains the APIs that are used to manage Applications as well as the Roles of an Application. Here are the APIs:
 
 * <<Create an Application>>
+* <<Create a New Application From an Existing Application>>
```

Comment:
You don't need to upper case words like `New`, `From` or `Existing`. Use sentence capitalization. 

`Create an Application from an existing Application`

Reply:
Ah - ok.  I was going off of our DevDocsReadMe:

> Headers should be title-case. (see https://titlecase.com/ to check if you would like. No caps on articles 👍)

By this measure, then `from` would be the only article that was incorrectly capitalized.  Is this correct?
 
Title Case result:
> == Create an New Application from an Existing Application